io.gitlab.Openlyst.doudou.yml

```diff
@@ -0,0 +1,123 @@
+app-id: io.gitlab.Openlyst.doudou
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: doudou
+
+finish-args:
+  # X11 + XShm access (fallback)
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Audio playback
+  - --socket=pulseaudio
+  # Network access (for streaming)
+  - --share=network
+  # OpenGL access
+  - --device=dri
+
+modules:
+  # FFmpeg - required by mpv
+  - name: ffmpeg
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --disable-doc
+      - --disable-programs
+      - --enable-gpl
+      - --enable-version3
+      - --enable-libvorbis
+      - --enable-libopus
+      - --enable-libvpx
+      - --enable-libmp3lame
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-7.0.2.tar.xz
+        sha256: 8646515b638a3ad303e23af6a3587734447cb8fc0a0c064ecdb8e95c4fd8b389
+    cleanup:
+      - /share/ffmpeg/examples
+
+  # libass - subtitle rendering
+  - name: libass
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
+        sha256: f0da0bbfba476c16ae3e1cfd862256d30915911f7abaa1b16ce62ee653192784
+
+  # libplacebo - required by mpv for GPU video output
+  - name: libplacebo
+    buildsystem: meson
+    config-opts:
+      - -Dvulkan=enabled
+      - -Dshaderc=disabled
+      - -Dglslang=disabled
+      - -Ddemos=false
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/libplacebo.git
+        tag: v6.338.2
+        commit: 64c1954570f1cd57f8570a57e51fb0249b57bb90
+
+  # MPV
+  - name: mpv
+    buildsystem: meson
+    config-opts:
+      - -Dlibmpv=true
+      - -Dcplayer=true
+      - -Dbuild-date=false
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/refs/tags/v0.39.0.tar.gz
+        sha256: 2ca92437affb62c2b559b4419ea4785c70d023590500e8a52e95ea3ab4554683
+    cleanup:
+      - /share/applications
+      - /share/icons
+      - /share/zsh
+
+  # Doudou application
+  - name: doudou
+    buildsystem: simple
+    build-commands:
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source